internal/mode/static/nginx/runtime/manager.go

```diff
@@ -54,23 +71,27 @@ type MetricsCollector interface {
 
 // ManagerImpl implements Manager.
 type ManagerImpl struct {
-	verifyClient     *verifyClient
+	verifyClient     verifyClient
 	metricsCollector MetricsCollector
-	ngxPlusClient    *ngxclient.NginxClient
+	ngxPlusClient    nginxPlusClient
 	logger           logr.Logger
+	process          processHandler
```

Comment:
I'd call the variable `processHandler` as well

Reply:
sure will do that